src/http/web_socket/close_code.cr

```diff
@@ -0,0 +1,26 @@
+enum HTTP::WebSocket::CloseCode
+  NORMAL_CLOSURE             = 1000
```

Comment:
I think we need to set a style guide for enum members, if only to avoid these discussions.

Reply:
I agree

I'm tired of the constant nitpicking style discussions from some people who frequent this project. There needs to be style guidelines so that we can rigidly accept anything which passes them.

Reply:
#8984

Reply:
@RX14 yeah, it seems that even core members don't know their code, have a look at `HTTP::Status` as an example, and you know... _code is art, everything is subjective, nothing matters, let it go_... xD

Reply:
I mean, bringing this up was kinda poking fun at your comment in the other PR to be fair :P

Reply:
Shit, did I write it first?! Where? xD

Reply:
.......... https://github.com/crystal-lang/crystal/pull/8975#discussion_r400498945

Reply:
Ah, that one, yeah... All I'm trying to do is be consistent with the rest of the codebase u know ;) Style guide would help in that for sure xD

Reply:
anyway, `PascalCase` won so I suggest that this enum be changed to use `PascalCase` like originally.

Makes this PR less of a breaking change as well.

Reply:
It's consistent with `HTTP::Status` members and could be changed with all the rest once the issue is settled.

Reply:
It's definitely been settled to `PascalCase`. No sense in adding more changes to the diff when we do change `HTTP::Status`.

Reply:
> It's definitely been settled to `PascalCase`.

I don't think so, since the issue (#7270) is still open.

> No sense in adding more changes to the diff when we do change `HTTP::Status`.

And what's the difference?

Reply:
It ain't worth spending more time on this, I've force-pushed the changes bringing back the `PascalCase`, now u can go and treat yourself with sth nice xD